project3/styles1.css

```diff
@@ -125,3 +133,87 @@ label {
     vertical-align: middle;
     text-align: center;
 }
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-apperance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 3px solid white;
+    height: 24px;
+    width: 24px;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: #71f79f;
+}
+
+.confirm {
+    margin-top: 100px;
+
+}
+
+.button-reset {
+    border: 3px;
+    height: 87px;
+    width: 206px;
+    background-color: #71f79f;
+    font-family: 'Teko', sans-serif;
+    font-size: 30px;
+    font-weight: 700;
+    position: relative;
+    text-align: center;
+    top: 45px;
+    left: 25%;
+}
+
+@media (min-width: 800px) {
+    .category {
+        width: 250px;
+        word-wrap: break-word;
+        position: relative;
+        top: 35px;
+        line-height: 1.1;
+    }
+
+    .gender {
+        position: relative;
+        left: 280px;
+        top: -345px;
+        line-height: 2;
+    }
+
+    .musslearm {
+        position: relative;
+        left: -200px;
+    }
+
+    .clock {
+        position: relative;
+        top: -215px;
+        left: 10px;
+    }
+
+    .bulb {
+        position: relative;
+        top: -432px;
+        left: 215px;
+    }
```

Comment:
It would be simpler and more adaptable to achieve this layout with `display: inline-block;`